### PR TITLE
Pud 122 record issuance of recall notification

### PR DESCRIPTION
**Pull request by shahnaazr**
https://dsdmoj.atlassian.net/browse/PUD-506

### CODE REVIEW

File: src/test/java/cucumber/steps/NavigationSteps.java [hunk 1]
```diff
@@ -302,18 +303,62 @@ public void submitCurrentPrison(String customer) {
...
+        );
+      //  getCurrentUrl();
+        theActorCalled(customer).remember("DOCUMENT_ID", textContent(DocumentPage.DOCUMENT_ID));
+        String documentId = theActorCalled(customer).recall("DOCUMENT_ID");
+        System.out.println("document id is : " + documentId);
+        theActorCalled(customer).attemptsTo(
+                Click.on(DocumentPage.DOWNLOAD_LINK),
+                SwitchToNewWindow.switchToNewTab()
+        );
+//   // userClicksOn(customer, RecallDetailsPage.RECALL_DOCUMENT_LINK_PART_A);
```
Comment (jonnywyatt) [inline src/test/java/cucumber/steps/NavigationSteps.java:342]: Best to. remove commented. code?

File: src/test/resources/features/managerecalls/ManageRecall.feature [hunk 2]
```diff
@@ -16,19 +16,21 @@ Feature: Manage Recalls
...
     When Maria confirms the recall length of 28 days
     And Maria submits the licence breach details
     And Maria submits the current prison details
+    And Maria records the issuance of recall notification
     Then Maria can see that the recall is authorised
     And Maria downloads the revocation order
     ### confirm the details captured during the recall assessment journey are displayed
-    When Maria begins to assess the recall that they have just booked
-    Then Maria is able to see the details captured during assessment
+    # When Maria begins to assess the recall that they have just booked
```
Comment (jonnywyatt) [inline src/test/resources/features/managerecalls/ManageRecall.feature:28]: Are these meant to be commented out?

File: src/test/java/cucumber/steps/NavigationSteps.java [hunk 3]
```diff
@@ -302,18 +303,62 @@ public void submitCurrentPrison(String customer) {
...
+                SwitchToNewWindow.switchToNewTab()
+
+        );
+      //  getCurrentUrl();
+        theActorCalled(customer).remember("DOCUMENT_ID", textContent(DocumentPage.DOCUMENT_ID));
+        String documentId = theActorCalled(customer).recall("DOCUMENT_ID");
+        System.out.println("document id is : " + documentId);
+        theActorCalled(customer).attemptsTo(
+                Click.on(DocumentPage.DOWNLOAD_LINK),
+                SwitchToNewWindow.switchToNewTab()
```
Comment (jonnywyatt) [inline src/test/java/cucumber/steps/NavigationSteps.java:340]: any way. we. can. download. it, or verify it's in the new tab?

File: src/test/resources/features/managerecalls/ManageRecall.feature (hunk 2 above)
Comment (mr-ken) [reply src/test/resources/features/managerecalls/ManageRecall.feature:28]: Yes - let's not leave commented out code (especially without any comments) ; it's difficult to know why the code is commented out.

**Review by mr-ken — APPROVED** [review]

**Pull request merged**